packages/datatrak-web/src/features/Questions/CheckboxQuestion.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Checkbox as BaseCheckbox } from '@tupaia/ui-components';
+import { SurveyQuestionInputProps } from '../../types';
+
+const Checkbox = styled(BaseCheckbox)`
+  margin: 0 0.56rem;
+  .MuiCheckbox-root {
+    padding: 0;
+    margin-right: 0.5rem;
+  }
+`;
+export const CheckboxQuestion = ({
+  id,
+  label,
+  name,
+  controllerProps,
+}: SurveyQuestionInputProps) => {
+  return (
+    <Checkbox
+      {...controllerProps}
```

Comment:
This is a good example of how explicitly writing out the controller props will make it simpler since 2 of the 4 props are overwritten anyway (name and onChange).